code/drasil-website/package.yaml

```diff
@@ -0,0 +1,48 @@
+name:                drasil-website
+version:             0.1.0.0
+github:              JacquesCarette/Drasil
+maintainer:          "Jacques Carette"
+synopsis:            Using the Drasil generators to create Drasil's website
+
+extra-source-files: []
+
+# Metadata used when publishing your package
+# category:            Web
+
+# To avoid duplicated efforts in documentation and dealing with the
+# complications of embedding Haddock markup inside cabal files, it is
+# common to point users to the README.md file.
+homepage:            https://jacquescarette.github.io/Drasil/
+description:         Please see the README on GitHub at <https://github.com/JacquesCarette/Drasil#readme>
+
+dependencies:
+- base >= 4.7 && < 5
+- lens
+- split
+- directory
+- drasil-data
+- drasil-database
+- drasil-docLang
+- drasil-gen
+- drasil-lang
+- drasil-printers
+- drasil-theory
+- drasil-utils
+
+ghc-options:
+- -Wall
+- -Wredundant-constraints
+- -threaded
+
+executables:
+  website:
+    main: Drasil.Website.Main
+    source-dirs: ./
+    other-modules:
+    - Drasil.Website.Website
+    - Drasil.Website.Analysis
+    - Drasil.Website.CaseStudy
+    - Drasil.Website.Documentation
+    - Drasil.Website.Example
+    - Drasil.Website.Graphs
+    - Drasil.Website.Introduction
```

Comment:
Was `hpack` not automatically picking these up?

Reply:
Yea, it wouldn't compile unless I put the extra module names in the package.yaml file.

Reply:
Oh, did you only have 1 or 2 listed in the package.yaml file? Does it still happen if you entirely remove the `other-modules` list section?

Reply:
Oh, yea I only had Drasil.Website.Website there before. I'll try removing them all and see if it does it automatically.